CondTools/SiPixel/plugins/SiPixelDynamicInefficiencyReader.cc

```diff
@@ -270,11 +270,13 @@ void SiPixelDynamicInefficiencyReader::analyze(const edm::Event& e, const edm::E
       mismatch++;
     }
     for (unsigned int i = 0; i < pu_det; i++) {
+      [[clang::suppress]]
```

Comment:
@iarspider , I think we do not need to suppress these here. As `pu_det` is size of [map_pufactor](https://github.com/cms-sw/cmssw/blob/master/CondTools/SiPixel/plugins/SiPixelDynamicInefficiencyReader.cc#L169) so just replacing the loop at line 201 should fix this warning
```
-    for (it_pu = map_pufactor.begin(); it_pu != map_pufactor.end(); it_pu++, pu_iterator++) {
+    for (it_pu = map_pufactor.begin(); pu_iterator < pu_det; it_pu++, pu_iterator++) {
```